hbase-client/src/main/java/org/apache/hadoop/hbase/client/RawAsyncHBaseAdmin.java

```diff
@@ -4146,4 +4148,21 @@ private void getProcedureResult(long procId, CompletableFuture<Void> future, int
           resp -> resp.hasRSGroupInfo() ? ProtobufUtil.toGroupInfo(resp.getRSGroupInfo()) : null)))
       .call();
   }
+
+  @Override
+  public CompletableFuture<Void> renameRSGroup(String oldName, String newName) {
```

Comment:
Would you like to provide any validation for both strings? e.g. non-empty

Reply:
I think it is ok to leave it as it is, server side will throw ConstraintException if both are empty. Other methods like addRSGroup neither checks non-empty.